src/components/Form.js

```diff
@@ -77,6 +77,9 @@ const propTypes = {
     /** Container styles */
     style: stylePropTypes,
 
+    /** Custom content to display in the footer after submit button */
+    footerContent: PropTypes.oneOfType([PropTypes.func, PropTypes.node]),
```

Comment:
When is the footer content a function?

Reply:
I took this from the children declaration on form component - https://github.com/Expensify/App/blob/8eb8cf9f458d590a35a865a3143eb1fdc9dfe916/src/components/Form.js#L37